File: framework/languages/java/e2e/SpotService/Client/src/main/java/systems/zlink/e2e/spotservice/client/Scenarios/SmE4Scenario.java
package systems.zlink.e2e.spotservice.client.Scenarios;


public final class SmE4Scenario extends SpotServiceScenarioContext {
    private SmE4Scenario(SpotServiceScenarioContext context) {
        super(context);
    }

    public static void run(SpotServiceScenarioContext context) {
        new SmE4Scenario(context).execute();
    }

    private void execute() {
        sleep(1200);
        System.out.println("scenario SM-E4 passed");

    }
}

File: framework/languages/java/e2e-kotlin/SpotService/Client/src/main/kotlin/systems/zlink/e2e/kotlin/spotservice/client/scenarios/SmE4Scenario.kt
package systems.zlink.e2e.kotlin.spotservice.client.scenarios

import systems.zlink.e2e.kotlin.spotservice.client.support.sleep

internal object SmE4Scenario {
    suspend fun run() {
        sleep(1200)
        println("scenario SM-E4 passed")
    }
}
